src/coreclr/inc/yieldprocessornormalized.h

```diff
@@ -20,6 +17,24 @@ FORCEINLINE void System_YieldProcessor() { YieldProcessor(); }
     T() = delete; \
     DISABLE_COPY(T)
 
+#ifdef FEATURE_NATIVEAOT
```

Comment:
It would be better to switch to the C/C++ standard lower-case min/max instead of adding this